packages/react-link/src/components/Link/useLinkState.ts

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { LinkState } from './Link.types';
+
+/**
+ * The useLink hook processes the Link draft state.
+ * @param draftState - Link draft state to mutate.
+ */
+export const useLinkState = (draftState: LinkState) => {
```

Comment:
I see no reason for `draftState`... I wonder why that's the naming convention ? 

```suggestion
export const useLinkState = (state: LinkState) => {
```

Reply:
Yeah, I think we should probably change that.